host_modules/file_service.py

```diff
@@ -0,0 +1,45 @@
+"""File stat handler"""
+
+from host_modules import host_service
+import subprocess
+
+MOD_NAME = 'file'
+EXIT_FAILURE = 1
+
+import os
+
+class FileService(host_service.HostModule):
+    """
+    Dbus endpoint that executes the file command
+    """
+    @host_service.method(host_service.bus_name(MOD_NAME), in_signature='s', out_signature='ia{ss}')
+    def get_file_stat(self, path):
```

Comment:
@isabelmsft looks like this is not used in this PR. is `get_file_stat` going to be used in future?

Reply:
That's right, `get_file_stat` is to be used by the File.Stat API as implemented in this PR: https://github.com/sonic-net/sonic-gnmi/pull/274/files